README.md

```diff
@@ -52,17 +67,56 @@ Its use should be automated in Composer scripts.
 }
 ```
 
-or
+This provides `composer strauss`, which does the following:
+
+1. The `sh -c` command tests if `bin/strauss.phar` exists, and if not, downloads it from [releases](https://github.com/BrianHenryIE/strauss/releases).
+2. Then `@php bin/strauss.phar` is run to prefix the namespaces.
+3. Ensure that composer's autoload map is updated.
+
+### As a dev dependency via composer (not recommended)
+
+If you prefer to include Strauss as a dev dependency, you can still do so. You mileage may vary when you include it this way.
```

Comment:
A bit vague. ~"This will not work if your plugin targets a version lower than Strauss's minimum, 7.4, or if Strauss happens to have dependency requirements whose versions constrains are incompatible with other packages you are installing"